server/api/cvrs.py

```diff
@@ -53,7 +53,7 @@ def process():
         for column, (contest_name, votes_allowed, contest_choice) in enumerate(
             interpretation_headers
         ):
-            contests_metadata[contest_name]["votes_allowed"] = votes_allowed
+            contests_metadata[contest_name]["votes_allowed"] = int(votes_allowed)
```

Comment:
why is it not an integer to start with?

Reply:
well it gets parsed on L44 above - I guess I could move the `int` up there, if that's what you're suggesting

Reply:
moved it